### PR TITLE
Rebrand [Rr]edis->[Vv]alkey in TLS and sentinel pages

### PR DESCRIPTION
**Pull request by daniel-house**
Changes Redis to Valkey. References to old versions are changed to Redis OSS.

### CODE REVIEW

File: docs/management/security/encryption.md [hunk 1]
```diff
@@ -121,11 +121,11 @@ Sentinel will be configured with `tls-port` if and only if `tls-replication` is
 
 Additional TLS configuration is available to control the choice of TLS protocol
 versions, ciphers and cipher suites, etc. Please consult the self documented
-`redis.conf` for more information.
+`valkey.conf` for more information.
 
 ### Performance considerations
 
-TLS adds a layer to the communication stack with overheads due to writing/reading to/from an SSL connection, encryption/decryption and integrity checks. Consequently, using TLS results in a decrease of the achievable throughput per Redis instance (for more information refer to this [discussion](https://github.com/redis/redis/issues/7595)). 
+TLS adds a layer to the communication stack with overheads due to writing/reading to/from an SSL connection, encryption/decryption and integrity checks. Consequently, using TLS results in a decrease of the achievable throughput per Valkey instance (for more information refer to this PLEASE_REVIEW{[discussion](https://github.com/redis/redis/issues/7595)}).
```
Comment (stockholmux) [inline docs/management/security/encryption.md:128]: This looks like ancient history - 4 years ago and a major version. I would drop the parenthetical.

WDYT @madolson?

File: docs/management/security/encryption.md [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
+description: Valkey TLS support
 aliases: [
     /topics/encryption,
     /docs/manual/security/encryption,
     /docs/manual/security/encryption.md
 ]
 ---
 
-SSL/TLS is supported by Redis starting with version 6 as an optional feature
+SSL/TLS is supported by Valkey starting with version 6 as an optional feature
```
Comment (zuiderkwast) [inline docs/management/security/encryption.md:13]: There is no Valkey version 6.

When refering to old redis version, write "Redis OSS". It is according to redis trademark policy.

File: docs/management/security/encryption.md (hunk 1 above)
Comment (zuiderkwast) [reply docs/management/security/encryption.md:128]: Agree. Let's remove that. Alternatively read that issue and write any relevant info in the docs and remove the link.

File: docs/management/security/encryption.md (hunk 2 above)
Comment (zuiderkwast) [reply docs/management/security/encryption.md:13]: In this case we can just remove "starting with version 6". Imo

Comment (daniel-house) [reply docs/management/security/encryption.md]: done

File: docs/management/sentinel.md [hunk 3]
```diff
@@ -847,7 +847,7 @@ configured with `requirepass`, the Sentinel configuration must include the
 Configuring Sentinel instances with authentication
 ---
 
-Sentinel instances themselves can be secured by requiring clients to authenticate via the `AUTH` command. Starting with Redis 6.2, the [Access Control List (ACL)](/topics/acl) is available, whereas previous versions (starting with Redis 5.0.1) support password-only authentication. 
+Sentinel instances themselves can be secured by requiring clients to authenticate via the `AUTH` command. PLEASE-REVIEW{Starting with Redis 6.2, the [Access Control List (ACL)](/topics/acl) is available, whereas previous versions (starting with Redis 5.0.1) support password-only authentication. }
```
Comment (zuiderkwast) [inline docs/management/sentinel.md:850]: If it's very old (like redis 2 or 3) it is always fine to just delete it. If it's newer and can be useful to anyone, we can keep it with the terminology "Redis OSS".

If you're not sure, it is always possible to describe old versions as Redis OSS. It is the terminology accepted by Redis Trademark Policy. There is no need to ask every time.

```suggestion
Sentinel instances themselves can be secured by requiring clients to authenticate via the `AUTH` command. Starting with Redis OSS 6.2, the [Access Control List (ACL)](/topics/acl) is available, whereas previous versions (starting with Redis OSS 5.0.1) support password-only authentication.
```

File: docs/management/sentinel.md [hunk 4]
```diff
@@ -1149,33 +1149,33 @@ We know that Sentinel 3 will get its configuration updated when the network
...
 replication and the discarding nature of the "virtual" merge function of the system. Note that this is not a limitation of Sentinel itself, and if you orchestrate the failover with a strongly consistent replicated state machine, the same properties will still apply. There are only two ways to avoid losing acknowledged writes:
 
 1. Use synchronous replication (and a proper consensus algorithm to run a replicated state machine).
 2. Use an eventually consistent system where different versions of the same object can be merged.
 
-Redis currently is not able to use any of the above systems, and is currently outside the development goals. However there are proxies implementing solution "2" on top of Redis stores such as SoundCloud [Roshi](https://github.com/soundcloud/roshi), or Netflix [Dynomite](https://github.com/Netflix/dynomite).
+Valkey currently is not able to use any of the above systems, and is currently outside the development goals. PLEASE-REVIEW{However there are proxies implementing solution "2" on top of Redis stores such as SoundCloud [Roshi](https://github.com/soundcloud/roshi), or Netflix [Dynomite](https://github.com/Netflix/dynomite).}
```
Comment (zuiderkwast) [inline docs/management/sentinel.md:1178]: I think this is interesting to keep. Here we need to rewrite it in a smarter way, not just search-replace.

What do you think about this suggestion?

```suggestion
Valkey (like it's predecessor Redis OSS) is currently not able to use any of the above systems, and is currently outside the development goals. However, there are proxies implementing solution "2" on top of Redis OSS stores such as SoundCloud [Roshi](https://github.com/soundcloud/roshi), or Netflix [Dynomite](https://github.com/Netflix/dynomite).
```

File: docs/management/sentinel.md [hunk 5]
```diff
@@ -1237,4 +1237,4 @@ API that many kernels offer. However it is not still clear if this is a good
 solution since the current system avoids issues in case the process is just
 suspended or not executed by the scheduler for a long time.
 
-**A note about the word slave used in this man page**: Starting with Redis 5, if not for backward compatibility, the Redis project no longer uses the word slave. Unfortunately in this command the word slave is part of the protocol, so we'll be able to remove such occurrences only when this API will be naturally deprecated.
+**A note about the word slave used in this man page**: PLEASE-REVIEW{Starting with Redis 5, if not for backward compatibility, the Redis project no longer uses the word slave. Unfortunately in this command the word slave is part of the protocol, so we'll be able to remove such occurrences only when this API will be naturally deprecated.}
```
Comment (zuiderkwast) [inline docs/management/sentinel.md:1240]: This note is in many doc pages. I think we should replace all of them in a separate PR.

File: docs/management/sentinel.md [hunk 6]
```diff
@@ -46,26 +46,26 @@ The current version of Sentinel is called **Sentinel 2**. It is a rewrite of
...
 
-A stable release of Redis Sentinel is shipped since Redis 2.8.
+PLEASE-REVIEW{A stable release of Valkey Sentinel is shipped since Redis 2.8.}
 
 New developments are performed in the *unstable* branch, and new features
 sometimes are back ported into the latest stable branch as soon as they are
 considered to be stable.
 
-Redis Sentinel version 1, shipped with Redis 2.6, is deprecated and should not be used.
+PLEASE-REVIEW{Redis Sentinel version 1, shipped with Redis 2.6, is deprecated and should not be used}.
```
[the remark this answers is not in the document]
Comment (daniel-house) [reply docs/management/sentinel.md:55]: removed

File: docs/management/sentinel.md [hunk 7]
```diff
@@ -639,13 +639,13 @@ Lastly, Sentinel also supports the `SUBSCRIBE`, `UNSUBSCRIBE`, `PSUBSCRIBE` and
 
 ### Reconfiguring Sentinel at Runtime
 
-Starting with Redis version 2.8.4, Sentinel provides an API in order to add, remove, or change the configuration of a given master. Note that if you have multiple sentinels you should apply the changes to all to your instances for Redis Sentinel to work properly. This means that changing the configuration of a single Sentinel does not automatically propagate the changes to the other Sentinels in the network.
+PLEASE-REVIEW{Starting with Redis version 2.8.4,} Sentinel provides an API in order to add, remove, or change the configuration of a given master. Note that if you have multiple sentinels you should apply the changes to all to your instances for Valkey Sentinel to work properly. This means that changing the configuration of a single Sentinel does not automatically propagate the changes to the other Sentinels in the network.
```
[the remark this answers is not in the document]
Comment (daniel-house) [reply docs/management/sentinel.md:642]: done

File: docs/management/sentinel.md [hunk 8]
```diff
@@ -657,9 +657,9 @@ As already stated, `SENTINEL SET` can be used to set all the configuration param
 
 Note that there is no equivalent GET command since `SENTINEL MASTER` provides all the configuration parameters in a simple to parse format (as a field/value pairs array).
 
-Starting with Redis version 6.2, Sentinel also allows getting and setting global configuration parameters which were only supported in the configuration file prior to that.
+PLEASE-REVIEW{Starting with Redis version 6.2,} Sentinel also allows getting and setting global configuration parameters which were only supported in the configuration file prior to that.
```
[the remark this answers is not in the document]
Comment (daniel-house) [reply docs/management/sentinel.md:660]: done

File: docs/management/sentinel.md (hunk 3 above)
Comment (daniel-house) [reply docs/management/sentinel.md:850]: done, thanks for the clarification

**Comment by zuiderkwast** [comment]
When we merge, by default we use the PR description as the commit message. It would be good if it describes the change, so we can just use that when merging.

File: docs/management/sentinel.md (hunk 5 above)
Comment (daniel-house) [reply docs/management/sentinel.md:1240]: So only a change to Redis OSS 5 in this PR.

If we are going to do all of them in a single PR, I would suggest having all such pages either link to an explanation or include one from a single file.  My preference would be a link to a page that also explains the movement from MASTER to PRIMARY.

Either a link or include would also let us improve the message in just one place.  It seems to me that "... when this API will be naturally deprecated." should be changed to "... MASTER and SLAVE have been deprecated in favor of PRIMARY and REPLICA and will be deleted when the community is ready to accept such a breaking change."

**Comment by daniel-house** [comment]
> When we merge, by default we use the PR description as the commit message. It would be good if it describes the change, so we can just use that when merging.

The commit message is "rebrand [Rr]edis->[Vv]alkey, docs/management/sentinel.md".  It states the purpose and the file affected.  What else do you need?

File: docs/management/sentinel.md [hunk 9]
```diff
@@ -794,31 +782,31 @@ fails and both S1 and S2 are available, S1 will be preferred.
...
-Where `<username>` and `<password>` are the username and password for accessing the group's instances. These credentials should be provisioned on all of the group's Redis instances with the minimal control permissions. For example:
+Where `<username>` and `<password>` are the username and password for accessing the group's instances. These credentials should be provisioned on all of the group's Valkey instances with the minimal control permissions. For example:
 
     127.0.0.1:6379> ACL SETUSER sentinel-user ON >somepassword allchannels +multi +slaveof +ping +exec +subscribe +config|rewrite +role +publish +info +client|setname +client|kill +script|kill
 
-### Redis password-only authentication
+### Valkey password-only authentication
 
-Until Redis 6, authentication is achieved using the following configuration directives:
+PLEASE-REVIEW{Until Redis 6,} authentication is achieved using the following configuration directives:
```
Comment (zuiderkwast) [inline docs/management/sentinel.md:809]:
```suggestion
Until Redis OSS 6, authentication was achieved using the following configuration directives:
```

**Review by zuiderkwast — APPROVED** [review]
Very good. I just found one more PLEASE-REVIEW tag.

**Comment by zuiderkwast** [comment]
> Oh, the PR description, "rebrand [Rr]edis->[Vv]alkey". I think that describes the change too. Please help me understand what's bothering you.

The description, the top comment, is `@madolson @zuiderkwast @stockholmux please review`. Some PRs were merged and they got this in the commit message. :grinning: For example this: https://github.com/valkey-io/valkey-doc/commit/93ebe821e264c32ddf5ad3d487e9e662b895fadc

File: docs/management/sentinel.md (hunk 9 above)
Comment (daniel-house) [reply docs/management/sentinel.md]: I found 2.  fixed

**Comment by zuiderkwast** [comment]
Look, now I changed the title to include also which pages where rebranded. This makes it a little more descriptive among all the other rebranding commits.

I changed the description to some simple explanation of what was changed. In the doc repo this is not very important, but in the code repo, we add links to the PRs in the release notes, so there it is more important to have a good description at the top.

**Comment by daniel-house** [comment]
> > Oh, the PR description, "rebrand [Rr]edis->[Vv]alkey". I think that describes the change too. Please help me understand what's bothering you.
> 
> The description, the top comment, is `@madolson @zuiderkwast @stockholmux please review`. Some PRs were merged and they got this in the commit message. 😀 For example this: [93ebe82](https://github.com/valkey-io/valkey-doc/commit/93ebe821e264c32ddf5ad3d487e9e662b895fadc)

I was actually told to put that "please review" comment in because I was not able to find a place in the UI for the first review.  Next time I will try to put it in a comment instead of in the description.  Unless you have a better idea?

**Comment by zuiderkwast** [comment]
Can you use the menu at the top-right to request review from people or is that only accessable for maintainers? I don't know. It looks like this:

image

Any way, sometimes it is not very clear what a PR is doing until after review and a lot of changes, so it is not very strange that we ask to update the description to describe what was actually implemented. This is only important when the PR is ready to merge. Before that, it can very well contains information which is useful for reviewers.

Thanks!

**Comment by daniel-house** [comment]
I will look for that menu on my next PR.  I couldn't find it last time, but I might have been looking in the wrong place.

**Comment by zuiderkwast** [comment]
It's probably only visible to maintainers.

Comment works fine.

**Pull request merged**